### ISSUE TITLE
[APP] Zoominfo

### ISSUE THREAD

**Issue by raphaelvarieras** [PipedreamHQ/pipedream#2982]
ZoomInfo

https://api-docs.zoominfo.com/#intro

Desired actions would be:
- look up contact
- look up company
- etc

**Comment by dylburger** [comment]
Thanks for the request! We've added this to [our integrations backlog](https://github.com/orgs/PipedreamHQ/projects/8/views/1), and we'll update this ticket with progress.

**Comment by malexanderlim** [comment]
@raphaelvarieras , do you have any API key that you've been using with their API? I applied to their partner program, but they are not going to approve the integration unless there are over 100 customers requesting it, and I was wondering if you're able to provide us with a key for development and testing purposes only. In addition, I was quoted ~$18K annual fee for an API key. 

We're not able to move forward at the moment without getting access to an API Key.

**Comment by raphaelvarieras** [comment]
I'm not able to help unfortunately - I don't have an API key or ability to generate one. That being said, ZoomInfo integration is no longer a priority for us.

**Comment by malexanderlim** [comment]
Got it, thanks - marking as blocked for now.

**Comment by raphaelvarieras** [comment]
So it seems there hasn't been much movement on this one. Would it be worth checking again with Zoominfo to see if there's interest in enabling this effort?

**Comment by malexanderlim** [comment]
We're in conversations with their team currently, as they require a minimum number of mutual customers using Zoominfo before they will consider granting access. To confirm, are you a current customer @raphaelvarieras?

**Comment by raphaelvarieras** [comment]
Yes, I can confirm that I am.

**Comment by malexanderlim** [comment]
So we could potentially build the integration, but in order to use it you'd need to provide your own API Key to connect your account - is this something you'd be able to do? I believe you mentioned before that you were unable to do so, which is a prerequisite to using the integration if we're able to build it. From my understanding, you may need to be on an Enterprise API plan to be able to generate an API Key.

We also integrate with **People Data Labs** and **Clearbit** which are great alternatives, and potentially lower barrier to entry as well if you are looking for other similar solutions.

**Comment by raphaelvarieras** [comment]
Yes, I can generate an API key for use with [this endpoint](https://api-docs.zoominfo.com/#authentication-methods:~:text=Public%20Key%20Infrastructure%20(PKI)%20authentication).

**Comment by sergio-eliot-rodriguez** [comment]
Hi @raphaelvarieras 

Recently Zoominfo was upvoted on #15964, but I don't know the user who requested because it shows a Pipedream service account to flag the issue on GitHub.

Are you able to help testing the base integration with Zoominfo?

**Comment by Lembergdk** [comment]
Is there any movement or update on this? Anything I can do to support this?

**Comment by sergio-eliot-rodriguez** [comment]
@Lembergdk thanks for reaching out. we are in contact with Zoominfo team to secure a sandbox for developing the integration.
